### PR TITLE
Bachmarks for runtime APIs including reports

### PR DESCRIPTION
**Pull request by bill828**
issue #17.

### CODE REVIEW

File: benchmarks/README.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Small|100 policies, 100 role policies, 10 users/role|8.5
+Medium|1K policies, 1K role policies, 10 users/role|8.6
+Large|10K policies, 10K role policies, 10 users/role|8.7
+Huge|100K policies, 100K role policies, 10 users/role|8.7
+LargeExp|10K policies with resource expression, 10K role policies, 10 users/role|43.0
+LargeCond|10K policies with condition, 10K role policies, 10 users/role|8.0
+
+以下表格展示了API `GetAllGrantedPermissions`的时间开销。
+
+用力|模型|时间开销 (μs/op)
```
Comment (dingwenfang) [inline benchmarks/README.zh-cn.md]: typo：用例

File: benchmarks/eval_b_test.go [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+	runTest(b, rc, filePath, true)
+}
+
+func BenchmarkHuge(b *testing.B) {
+	filePath, err := writePolicyFile(b.Name(), 100000, 10, simplePolicyWriter, simpleRolePolicyWriter)
+	if err != nil {
+		b.Fatalf("unable to write to policy file because of error %s", err)
+	}
+	defer os.Remove(filePath)
```
Comment (dingwenfang) [inline benchmarks/eval_b_test.go]: So the benchmark contains the time of removing file?

Comment (bill828) [reply benchmarks/eval_b_test.go]: calls b.StopTimer() at the end of each case.

**Comment by dingwenfang** [comment]
I'm OK with the change now.

**Pull request merged**